tests/integration_tests/ddl_manager/data/prepare.sql

```diff
@@ -98,8 +98,37 @@ CREATE TABLE t10 (
 
 DROP TABLE t9;
 
+
+---Test TiCDC can handle DDLs that across databases
+---Please note that we should write the database name explicitly in the DDLs
+
+drop database if exists `across_db_1`;
+create database `across_db_1`;
+
+drop database if exists `across_db_2`;
+create database `across_db_2`;
+
+use `across_db_1`;
+
+CREATE TABLE t1 (
+                    id INT AUTO_INCREMENT PRIMARY KEY,
+                    val INT DEFAULT 0,
+                    col0 INT NOT NULL
+);
+
+RENAME TABLE `across_db_1.t1` TO `across_db_2.t1`;
```

Comment:
add a negative case, such as " rename table t1 to db2.t1"

Reply:
If it is added, the test will failed. I can add it in the comment.